src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,500;1,300&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        alt="Logo MOYO"
+        class="logo_moyo"
+      >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">SAMSUNG</a>
```

Comment:
Don't set capital letters for all links. Better to style them via css
```suggestion
            <a href="#samsung" class="nav__link">samsung</a>
```